projects/storefrontapp/src/app/app.module.ts

```diff
@@ -93,6 +95,7 @@ if (!environment.production) {
       },
     }),
     RulebasedConfiguratorRootModule,
+    CpqConfiguratorRootModule,
```

Comment:
Now we should import `CpqConfiguratorRootModule` in app module only conditionally, because it forces early login. That would break core e2e tests. SPA CI wont pass.

Take as an example CDC, see latest `develop`:
- environment.ts and environment.prod.ts https://github.com/SAP/spartacus/blob/84a983fe421a4e39ff43a4208ca5d104f65f9ab7/projects/storefrontapp/src/environments/environment.ts#L22
- .env-cmdrc https://github.com/SAP/spartacus/blob/84a983fe421a4e39ff43a4208ca5d104f65f9ab7/.env-cmdrc#L8-L11
- SpartacusFeaturesModule https://github.com/SAP/spartacus/blob/84a983fe421a4e39ff43a4208ca5d104f65f9ab7/projects/storefrontapp/src/app/spartacus/spartacus-features.module.ts#L95-L97

Reply:
Ok. I was thinking to _not_ include the CPQ root module (meaning to remove before merge) in app.module.ts, and to provide a specific app.module.ts in our CI. But agree, your proposal is better. Will take a look.